Source/Managers/LuaMan.h

```diff
@@ -8,6 +8,8 @@
 
 #include "BS_thread_pool.hpp"
 
+#include <optional>
```

Comment:
Is this necessary? Where are we using this?

Reply:
Good catch! Removed that vistigial #include